BUILDING.md

```diff
@@ -300,53 +300,55 @@ To install this version of Node.js into a system directory:
 
 To verify the build:
 
-```console
-$ make test-only
+```text
+make test-only
 ```
 
 At this point, you are ready to make code changes and re-run the tests.
 
-If you are running tests before submitting a pull request, the recommended
-command is:
+If you are running tests before submitting a pull request, use:
 
-```console
-$ make -j4 test
+```text
+make test
 ```
 
 `make -j4 test` does a full check on the codebase, including running linters and
 documentation tests.
 
-Make sure the linter does not report any issues and that all tests pass. Please
-do not submit patches that fail either check.
-
-If you want to run the linter without running tests, use
+To run the linter without running tests, use
 `make lint`/`vcbuild lint`. It will lint JavaScript, C++, and Markdown files.
 
 If you are updating tests and want to run tests in a single test file
 (e.g. `test/parallel/test-stream2-transform.js`):
 
 ```text
-$ python tools/test.py test/parallel/test-stream2-transform.js
+python tools/test.py test/parallel/test-stream2-transform.js
```

Comment:
You would have to still document how to run on Windows.

Reply:
On Windows, isn't the executable named `py3.exe`, not `python`? I might be wrong as I haven't used Windows in a while.

Reply:
According to https://docs.python.org/3/faq/windows.html#how-do-i-make-python-scripts-executable:

```
On Windows, the standard Python installer already associates the .py extension with a file type (Python.File) and gives that file type an open command that runs the interpreter (`D:\Program Files\Python\python.exe "%1" %*`). This is enough to make scripts executable from the command prompt as ‘foo.py’.
```

So I think we can remove it after all. I had assumed that this was needed for Windows because Windows would not understand the `#!` on the first line of the file. But it looks like we can remove it after all.

Reply:
Unfortunately I think it depends on how you install Python. I have seen reports of using `py3.exe` but on my Windows 10 install:
```console
C:\Users\rlau>where python3
C:\Users\rlau\AppData\Local\Microsoft\WindowsApps\python3.exe

C:\Users\rlau>where python2
INFO: Could not find files for the given pattern(s).

C:\Users\rlau>where python
C:\Users\rlau\AppData\Local\Microsoft\WindowsApps\python.exe

C:\Users\rlau>where py3
INFO: Could not find files for the given pattern(s).

C:\Users\rlau>
```
I don't remember whether I installed Python manually or as part of Visual Studio 2019 (or it may have even been installed by my employer via Software Center deployment).